cli/src/cmd_auth.rs

```diff
@@ -314,134 +314,65 @@ impl CmdAuthLogin {
     }
 }
 
-/// Log out of an Oxide host.
+/// Removes authentication information saved in the hosts.toml file.
 ///
-/// This command removes the authentication configuration for a host either specified
-/// interactively or via `--host`.
-///
-///     $ oxide auth logout
-///     # => select what host to log out of via a prompt
-///
-///     $ oxide auth logout --host oxide.internal
-///     # => log out of specified host
+/// This command does not invalidate any tokens from the hosts, nor removes
+/// any authentication information saved in environment variables.
```

Comment:
```suggestion
/// This command does not invalidate any tokens from the hosts, nor remove
/// any authentication information saved in environment variables.
```

"nor remove any authentication information saved in environment variables." I'm not sure I understand what this means

Reply:
Changed the phrasing, let me know what you think :)